### DISCUSSION TITLE
Bruno Golden Edition - git add remote via Git UI

### DISCUSSION THREAD

**Discussion by qasimmahmood95** [usebruno/bruno#2998] [Q&A]
So there's currently the ability to commit, push, pull, etc from the Git UI, but one thing I can't figure out is how to add a remote 
I can obviously do it from the command line, but I was wondering if there's a way to do it directly from the Git UI?

**Comment by qasimmahmood95** [comment]
![image](https://github.com/user-attachments/assets/c2d7ea37-ce44-4308-ac15-9ae31892cea3)
I did find this under 'Checkout Branch', but clicking 'Choose Remote' doesn't seem to do anything

This is what I get when I try to either push or pull - no option to choose or add a remote
![image](https://github.com/user-attachments/assets/65dc2d02-1c18-411b-b792-f039e64890a6)

I'm on Windows 11 btw

**Comment by anusreesubash** [comment]
Choose Remote lets you choose already added remotes.

Currently, Bruno doesn't support adding new remotes directly, but this feature will be available in about a week.

In the meantime, you can add a remote via the terminal by following these steps:

1. Open your terminal: Navigate to your project's root directory using the terminal.

2. Add a new remote: Use the git remote add command to add a new remote. The syntax is:
   **git remote add remote-name remote-url**
   Example: git remote add origin https://github.com/username/repository.git
